models/salesforce__manager_performance.sql

```diff
@@ -7,20 +7,28 @@ with opportunity_aggregation_by_owner as (
 
     select *
     from {{ var('user') }}
-  
+
+-- If using user_role table, the following will be included, otherwise it will not.
+{% if var('salesforce__user_role_enabled', True) %}
 ), user_role as (
 
     select *
     from {{ var('user_role') }}
-  
+{% endif %}
+
 )
 
 select 
   coalesce(manager.user_id, 'No Manager Assigned') as manager_id,  
   coalesce(manager.user_name, 'No Manager Assigned') as manager_name,
   manager.city as manager_city,
   manager.state as manager_state,
-  user_role.user_role_name as manager_position,
+
+  -- If using user_role table, the following will be included, otherwise it will not.
+  {% if var('salesforce__user_role_enabled', True) %} 
+  user_role.user_role_name as manager_position, 
+  {% endif %}
```

Comment:
Even though this field is disabled here, is is still included in the `group by` statement on line 76 and causes the query to fail.

You will want to expand the conditional on lines 72-74 use a reduced group by if the user disables the `user_role` model and fields.